README.md

```diff
@@ -8,7 +8,8 @@ See https://github.com/cclib/cclib/issues/1395 for more information.
 - We do not expect to make any further tagged or versioned releases on the `master` branch.
 - This message will disappear when the final release of 2.0, after any alphas/release candidates/etc. is made.
 
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8280878.svg)](https://doi.org/10.5281/zenodo.8280878)
+[![Paper](https://img.shields.io/badge/Paper-10.1063/5.0216778-blue)](https://doi.org/10.1063/5.0216778)
```

Comment:
Perhaps also put the old paper here for the records.

Reply:
I left the old paper in the docs. Would you like it in the badges too?

Reply:
I'm ok with it only in the docs.